packages/web-react/src/components/TextArea/stories/TextAreaDisabled.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import TextArea from '../TextArea';
+
+// @see: https://github.com/storybookjs/storybook/issues/8104#issuecomment-932310244
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const Story = (props: unknown) => (
```

Comment:
@tomassychra @vmilersky – Guys, look at the comment on line 4. I think we shouldn't remove it.

Reply:
The comment is still applicable and removing `props` breaks **show code** functionality in Storybook, eg. it will not be able to show the code of the story.

Reply:
I'll marked all of these comments as resolved, cc @tomassychra.